### PR TITLE
[ja] DL tips and tricks

### PR DESCRIPTION
**Pull request by Kamulau**

### CODE REVIEW

**Comment by shervinea** [comment]
Great work @Kamulau, thank you for your help!

**Comment by shervinea** [comment]
Perfect @Kamulau, let's now start the review process! Please feel free to invite anyone who you know might want to help out.

**Comment by Kamulau** [comment]
Thanks for the comments! I'll be sure to invite anyone who might want to help out.

**Comment by Kamulau** [comment]
@yoshiyukinakai 
Thanks so much for your comments! Should I add those revisions on my end, or would you like to submit a pull request with the changes to this branch?

**Comment by shervinea** [comment]
@Kamulau please feel free to directly include reviewers suggestions in your revisions. 
At the end of the process, everyone involved will be added to the `CONTRIBUTORS` file regardless of whether they actually pushed a commit.

**Comment by Kamulau** [comment]
@shervinea 
Okay, great, thanks!

File: ja/deep-learning-tips-and-tricks.md [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+**33. Xavier initialization ― Instead of initializing the weights in a purely random manner, Xavier initialization enables to have initial weights that take into account characteristics that are unique to the architecture.**
+
+&#10230;Xavier初期化 - ランダムで重みを初期化するよりもむしろニューラルネットワークのアーキテクチャの特徴を用いて重みを初期化する手法です。
+
+<br>
+
+
+**34. Transfer learning ― Training a deep learning model requires a lot of data and more importantly a lot of time. It is often useful to take advantage of pre-trained weights on huge datasets that took days/weeks to train, and leverage it towards our use case. Depending on how much data we have at hand, here are the different ways to leverage this:**
+
+&#10230;転移学習 - 深層学習のモデルを学習させるには大量のデータと、それ以上に時間が必要です。膨大なデータで数日・数週間をかけて学習済みのモデルを利用し、自分のユースケースに活かせることが多いです。データの量次第では、以下の生かす方法があります。
```
Comment (Hiroki-Mori360) [inline ja/deep-learning-tips-and-tricks.md]:
```suggestion
&#10230;転移学習 - 深層学習のモデルを学習させるには大量のデータと何よりも時間が必要です。膨大なデータセットから数日・数週間をかけて構築した学習済みモデルを利用し、自身のユースケースに活かすことは有益であることが多いです。手元にあるデータ量次第ではありますが、これを利用する以下の方法があります。
```

File: ja/deep-learning-tips-and-tricks.md [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
+
+&#10230;収束の最適化
+
+<br>
+
+
+**39. Learning rate ― The learning rate, often noted α or sometimes η, indicates at which pace the weights get updated. It can be fixed or adaptively changed. The current most popular method is called Adam, which is a method that adapts the learning rate.
+**
+
+&#10230;学習率 - αやηとよく表記される学習率とは、重みの更新の速さを表現します。固定で指定するか、もしくは適応的に変えます。もっとも多用されている手法は、適切に学習率を変える Adam です。
```
Comment (Hiroki-Mori360) [inline ja/deep-learning-tips-and-tricks.md]:
```suggestion
&#10230;学習率 - 多くの場合αや時々ηと表記される学習率とは、重みの更新速度を表しています。学習率は固定することもできる上に、適応的に変更することもできます。現在もっとも使用される手法は、学習率を適切に調整するAdamと呼ばれる手法です。
```

File: ja/deep-learning-tips-and-tricks.md [hunk 3]
```diff
@@ -0,0 +1,457 @@
...
+
+<br>
+
+
+**40. Adaptive learning rates ― Letting the learning rate vary when training a model can reduce the training time and improve the numerical optimal solution. While Adam optimizer is the most commonly used technique, others can also be useful. They are summed up in the table below:**
+
+&#10230;適応学習率法 - 学習時間の短縮や精度の向上のために学習率を変更することです。Adamがもっとも多用されている手法だが、他の手法も役に立つことがあります。適応学習率法を下記の表にまとめました。
```
Comment (Hiroki-Mori360) [inline ja/deep-learning-tips-and-tricks.md]:
```suggestion
&#10230;適応学習率法 - モデルを学習させる際に学習率を変動させることで、学習時間の短縮や精度の向上につながります。Adamがもっとも一般的に使用されている手法ではあるが、他の手法も役立つことがあります。それらの手法を下記の表にまとめました。
```

File: ja/deep-learning-tips-and-tricks.md [hunk 4]
```diff
@@ -0,0 +1,457 @@
...
+**14. [Color shift, Noise addition, Information loss, Contrast change]**
+
+&#10230;カラーシフト、ノイズの付加、情報損失、コントラスト（鮮やかさ）の修正
+
+<br>
+
+
+**15. [Nuances of RGB is slightly changed, Captures noise that can occur with light exposure, Addition of noise, More tolerance to quality variation of inputs, Parts of image ignored, Mimics potential loss of parts of image, Luminosity changes, Controls difference in exposition due to time of day]**
+
+&#10230;RGBのわずかな修正、照らされ方によるノイズを捉える、ノイズの付加、入力画像の画質のバリエーションに対する耐性の増加、画像の一部を不使用、画像の一部がないときを真似る、明るさの変化、時間によるコントラストをコントロール
```
Comment (yoshiyukinakai) [inline ja/deep-learning-tips-and-tricks.md]:
```suggestion
&#10230;RGBのわずかな修正、照らされ方によるノイズを捉える、ノイズの付加、入力画像の品質のばらつきへの耐性の強化、画像の一部を無視、画像の一部が欠ける可能性を再現する、明るさの変化、時刻による露出の違いをコントロールする
```

File: ja/deep-learning-tips-and-tricks.md [hunk 5]
```diff
@@ -0,0 +1,457 @@
...
+**16. Remark: data is usually augmented on the fly during training.**
+
+&#10230;備考：データ拡張は基本的には学習時に行う。
+
+<br>
+
+
+**17. Batch normalization ― It is a step of hyperparameter γ,β that normalizes the batch {xi}. By noting μB,σ2B the mean and variance of that we want to correct to the batch, it is done as follows:**
+
+&#10230;batch normalization - ハイパーパラメータ γ、β のステップがバッチ {xi}を正規化します。平均と分散をμB,σ2Bと表記すると、以下で行えます。
```
Comment (yoshiyukinakai) [inline ja/deep-learning-tips-and-tricks.md]:
```suggestion
&#10230;batch normalization - ハイパーパラメータ γ、β によってバッチ {xi} を正規化するステップです。修正を加えたいバッチの平均と分散をμB,σ2Bと表記すると、以下のように行えます。
```

File: ja/deep-learning-tips-and-tricks.md [hunk 6]
```diff
@@ -0,0 +1,457 @@
...
+
+<br>
+
+
+**18. It is usually done after a fully connected/convolutional layer and before a non-linearity layer and aims at allowing higher learning rates and reducing the strong dependence on initialization.**
+
+&#10230;高い学習率を利用可能にするのと初期化への依存を減らすのが目的で基本的には全結合層・畳み込み層のあとで非線形層の前に行います。
```
Comment (yoshiyukinakai) [inline ja/deep-learning-tips-and-tricks.md]:
```suggestion
&#10230;より高い学習率を可能にし初期化への強い依存を減らすことを目的として、基本的には全結合層・畳み込み層のあとで非線形層の前に行います。
```

**Comment by Kamulau** [comment]
@Hiroki-Mori360 @yoshiyukinakai 

Thanks for your suggestions!! I have committed the changes. :)

**Comment by shervinea** [comment]
Thank you @Hiroki-Mori360 and @yoshiyukinakai for your suggestions and thank you @Kamulau for your prompt modifications! No worries about commit messages, feel free to just use them at discretion.

@Kamulau Would you say your work is now ready for merge?

**Comment by Kamulau** [comment]
@shervinea I just took another look through it, I think it looks ready for merging!

**Comment by shervinea** [comment]
Thanks for the feedback, glad to know! Will move forward with the merge and adding contributors to the list.

**Pull request merged**